USER_GUIDE.md

```diff
@@ -0,0 +1,26 @@
+# OpenSearch Benchmark Workloads User Guide
+
+OpenSearch Benchmark (OSB) comes packaged with official workloads that can be found throughout this repository. This guide provides a brief overview of the organizsation of this repository as well as how OSB uses these workloads.
+
+### Contents
+- [What do branches represent](#what-do-the-numbered-branches-represent)
+- [How does OSB select which branch to use](#how-does-osb-select-which-branch-to-use)
+- [Force OSB to use a Specific Branch](#force-osb-to-use-a-specific-branch)
```

Comment:
Did you plan to add more to this guide? Not sure if these links are all that helpful otherwise haha

Reply:
That's a good point, I just added it in case we ever expand upon this guide. I'm fine with keeping it or discarding it.